apps/protocol-api/src/prisma/resolvers/Contribution.ts

```diff
@@ -267,6 +267,22 @@ export class ContributionCountByDate {
   name: string;
 }
 
+@TypeGraphQL.ObjectType('ContributionCountByUser', { isAbstract: true })
+export class ContributionCountByUser {
+  @TypeGraphQL.Field(_type => Number)
+  count: number;
+
+  @TypeGraphQL.Field(_type => Number, {
+    nullable: true
+  })
+  user_id?: number;
+
+  @TypeGraphQL.Field(_type => String, {
+    nullable: true
+  })
+  user_name?: string;
```

Comment:
probably should rename this to display_name to avoid any confusion with name and username.